### ISSUE TITLE
Redis url is not honored even when specified in the config

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-base/RADAR-IoT#15]
The python module does not honor the redis URL mentioned in the config and always uses localhost (the default). Investigate and fix this.

**Comment by Hsankesara** [comment]
Hi Yatharth, I'd like to work on this issue, reproduce the bug and fix it.

**Comment by yatharthranjan** [comment]
Hi @Hsankesara , sure. To reproduce it, set the Redis URL to something different in the config file. It will still take it up as localhost no matter what is configured. It could be using one of the default values for parameters in init (possibly in the Connection and Publisher classes).

**Comment by Hsankesara** [comment]
Thanks @yatharthranjan, I changed the host in redis_publisher in config.yaml from localhost  to xyz but I got connection error and it didn't use localhost for connection.

**Comment by yatharthranjan** [comment]
Yes, that is right, this happens 2 times at startup, the first time it takes the correct URL from the config and hence you see the error, but if you get past the error, this will happen a second time, and this time it will use localhost. Can you try that?
You can add a new host binding on your system by editing `/etc/hosts` file.
